docs/frontmatter.md

```diff
@@ -110,6 +110,17 @@ title: My beautiful page
 ---
 ```
 
+## `weight`
+
+Affects the order a page is displayed in the sidebar navigation tree. Lower
```

Comment:
This confused me a bit because a higher weight in search results often means that things will end up higher in the results. What would you think of flipping the logic and naming this `navigation_priority`?

Big 👍 for documenting the option.

Reply:
Sure, that seems more explicit. I'll ask @jonathanglassman for his thoughts.

Reply:
I'm happy with either. "Weight = heavier sinks to the bottom" made sense to me, but navigation priority also makes sense. I would teach it as "smaller numbers = higher up the TOC" so as long as people don't think that higher numbers mean a higher priority in the TOC, we should be ok. If I had to make a choice I would keep it as is.

Reply:
Yeah I see your point, it would be nice to have something that just makes sense without an explanation. It's hard to come up with a scale that's intuitive.

Jon and I had a quick chat this morning, and decided to stick with weight for now and he really connected with the analogy. However, we want to push this out wider to other writers and see if they get confused by this.

Reply:
I haven't had time to look at the PR as a whole yet, but just wanted to "weigh in" on this discussion... 

I would interpret weight in the same way as @tijmenb as in more weight = more important = more prominent.

I would call it something like `navigation_order`.